R/parameters.R

```diff
@@ -180,9 +180,8 @@ generate_parameters <- function(
                          D0 = R0,
                          trans_increase = trans_increase_array,
                          trans_profile = trans_profile_array,
-                         beta_list = beta,
-                         beta = beta[1], # When odin code is updated, set this list
-                         beta_dates = beta_dates,
+                         beta_y = beta,
+                         beta_t = beta_dates,
```

Comment:
Just double-checking the definition at https://github.com/mrc-ide/sircovid/blob/master/R/parameters.R#L100 is compatible with the rest of the odin code. Unix time stamps differences (days) from the first date entered here

Reply:
I think that this is correct but @MJomaba needs to confirm. I feel we're very much at risk of off-by-one errors here, but the conversion looked correct to within one day to me

Reply:
I think it is fine